adafruit_magtag/network.py

```diff
@@ -68,13 +73,13 @@ def __init__(
         )
 
     @property
-    def enabled(self):
+    def enabled(self) -> bool:
         """
         Get or Set whether the WiFi is enabled
 
         """
         return self._wifi.enabled
 
     @enabled.setter
-    def enabled(self, value):
+    def enabled(self, value: Any) -> None:
```

Comment:
I know the function uses `bool()` but I think the intention is to actually use `bool` for `value` so we should type it as such.